Jackfruit.IncrementalGenerator/Jackfruit.IncrementalGenerator.csproj

```diff
@@ -34,8 +34,35 @@
 
 	<ItemGroup>
 		<ProjectReference Include="../Jackfruit.Runtime/Jackfruit.Runtime.csproj" />
-		<ProjectReference Include="../Jackfruit.CodeOutput/Jackfruit.CodeOutput.csproj" PrivateAssets="all"/>
-		<ProjectReference Include="../Jackfruit.Common/Jackfruit.Common.csproj" PrivateAssets="all"/>
+		<ProjectReference Include="../Jackfruit.CodeOutput/Jackfruit.CodeOutput.csproj" PrivateAssets="all" />
+		<ProjectReference Include="../Jackfruit.Common/Jackfruit.Common.csproj" PrivateAssets="all" />
+	</ItemGroup>
+
+	<Target
```

Comment:
this is patterned after what we have to do for MSBuild targets to collect reference output